src/collection/components/modals/ShareCollectionModal.tsx

```diff
@@ -95,10 +95,8 @@ const ShareCollectionModal: FunctionComponent<ShareCollectionModalProps> = ({
 
 			// Public status changed => log as event
 			trackEvents({
-				event_object: {
-					type: 'collection',
-					identifier: String(collection.id),
-				},
+				event_object: String(collection.id),
```

Comment:
Ik vind `${collection.id}` persoonlijk properder maar misschien ligt dat meer aan mij.

Reply:
dat doet me zo denken aan:
```
function toString(num: number) {
  return '' + prop;
}
```

Reply:
volgens dit article is String() en num.toString() het snelst
en is String() beter aangezien het niet faalt als num undefined is
https://stackoverflow.com/questions/5765398/whats-the-best-way-to-convert-a-number-to-a-string-in-javascript

Reply:
TIL :)